packages/venia-ui/lib/util/formValidators.js

```diff
@@ -108,3 +108,9 @@ export const isNotEqualToField = (value, values, fieldKey) => {
         ? SUCCESS
         : `${fieldKey} must be different`;
 };
+
+export const isValidEmail = value => {
```

Comment:
I'm surprised this has not been implemented before. Which makes me think there is a reason. @jimbo do you recall us ever having an form validator for emails?

Reply:
I was thinking about this validate many time when create an account

Reply:
One existed, but was purposely removed. Please do not re-add. I'll try to dig up the old discussion, but it was similar to the points in this article.

https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643

Reply:
Ok I removed that